src/ad-engine/providers/gpt-provider.ts

```diff
@@ -186,20 +186,23 @@ export class GptProvider {
 
 	destroySlots(slotNames) {
 		const allSlots = window.googletag.pubads().getSlots();
-		const slotsToDestroy =
-			slotNames && slotNames.length
-				? allSlots.filter((slot) => {
-					const slotId = slot.getSlotElementId();
-
-					if (!slotId) {
-						logger(logGroup, 'destroySlots', "slot doesn't return element id", slot);
-					} else if (slotNames.indexOf(slotId) > -1) {
-						return true;
-					}
-
-					return false;
-				  })
-				: allSlots;
+		let slotsToDestroy;
```

Comment:
```suggestion
		let slotsToDestroy = allSlots;
```
by default? (without else statement)